build.gradle

```diff
@@ -787,9 +787,7 @@ project(':core') {
     testCompile libs.apachedsLdifPartition
     testCompile libs.apachedsMavibotPartition
     testCompile libs.apachedsJdbmPartition
-    testCompile libs.junitJupiterApi
-    testCompile libs.junitVintageEngine
-    testCompile libs.scalatest
```

Comment:
We should update `dependencies.gradle` to remove `scalatest` too, right?

Reply:
```streams-scala``` still depends on that so #9858 will remove the dependency.